<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accounts_wizard_base</class>
 <widget class="QWidget" name="accounts_wizard_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>218</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&amp;User name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>kcfg_username</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel1_2">
     <property name="text">
      <string>&amp;Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>kcfg_password</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KLineEdit" name="kcfg_password">
     <property name="toolTip">
      <string>The password associated to your user name for the service</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="passwordMode">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="textLabel2_2">
     <property name="text">
      <string>&amp;Direction:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>kcfg_direction</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KComboBox" name="kcfg_direction">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Left to Right</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right to Left</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KLineEdit" name="kcfg_username">
     <property name="toolTip">
      <string>The name you use to connect to the service</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note:&lt;/span&gt; The Alias must be Unique.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Service:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_service</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="kcfg_service">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Twitter.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Identi.ca</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Laconica</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="kcfg_alias">
     <property name="toolTip">
      <string>The account alias</string>
     </property>
     <property name="whatsThis">
      <string>The alias is the name you want to give to your account. It should be unique. You can have several connections to the same service so the alias lets you give them names.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Alias:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_alias</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblHomepage">
     <property name="text">
      <string>Homepage:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KLineEdit" name="kcfg_homepage">
     <property name="toolTip">
      <string>Enter your custom Laconica homepage</string>
     </property>
     <property name="text">
      <string comment="KDE::DoNotExtract">http://</string>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_alias</tabstop>
  <tabstop>kcfg_service</tabstop>
  <tabstop>kcfg_homepage</tabstop>
  <tabstop>kcfg_username</tabstop>
  <tabstop>kcfg_password</tabstop>
  <tabstop>kcfg_direction</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
